MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -246,7 +246,6 @@ private void initComponents() {
         resourceMap = ResourceBundle.getBundle("mekhq.resources.CampaignGUI", new EncodeControl()); //$NON-NLS-1$
 
         frame = new JFrame("MekHQ"); //$NON-NLS-1$
-        MekHQ.setWindow(frame);
```

Comment:
?

Reply:
Unused variable, so cleaned it out.

Reply:
It does not, made sure of it when I did so.